runner.go

```diff
@@ -389,19 +389,27 @@ func (r *LocalRunner) ContainerRunner(step Step, network Network) func() error {
 // NetworkCreator returns a function to create the given network.
 func (r *LocalRunner) NetworkCreator(network Network) func() error {
 	return func() error {
+		// Check if network already exists
 		if Verbose {
-			log.Printf("Create network '%s'", network)
+			log.Printf("Check if network '%s' already exists", network)
 		}
-		if _, err := r.Output([]string{"network", "create", string(network)}); err != nil {
-			if err, ok := err.(*exec.ExitError); ok {
-				if strings.TrimSpace(string(err.Stderr)) != fmt.Sprintf("Error response from daemon: network with name %s already exists", string(network)) {
-					return err
-				}
-				if Verbose {
-					log.Printf("Network '%s' already exists", network)
-				}
+		out, err := r.Output([]string{"network", "ls", "--format", "{{.Name}}", "--filter", fmt.Sprintf("name=%s$", network)})
```

Comment:
As for the other PR these calls likely appear in multiple places so maybe there should be a wrapper function.

Reply:
It is currently the only occurrence but I'll move it into its own function

Reply:
Just try to keep it consistent with the other function you created for the other PR then feel free to merge.